doc/whats-new.rst

```diff
@@ -15,6 +15,14 @@ What's New
 
 .. _whats-new.0.10.0:
 
+Bug fixes
+~~~~~~~~~
+
+- Suppress warning in Ipython autocompletion, related to the deprecation
+  of ``.T`` attributes. (:issue:`1675`).
+  By `Keisuke Fujii <https://github.com/fujiisoup>`_
```

Comment:
Can you add this to a separate section below, something like "Bug fixes since rc1?